doctrine/mongodb-odm-bundle/3.3/etc/packages/doctrine_mongodb.yaml

```diff
@@ -0,0 +1,18 @@
+doctrine_mongodb:
+    auto_generate_proxy_classes: "%kernel.debug%"
+    auto_generate_hydrator_classes: "%kernel.debug%"
```

Comment:
I suggest using single quotes. Double quotes in YAML require escaping lots of things.